stdlib/crypto/src/main/ballerina/src/crypto/crypto.bal

```diff
@@ -218,121 +229,132 @@ function externCrc32b(byte[] input) returns handle = @java:Method {
+# ```ballerina byte[]|crypto:Error signature = crypto:signRsaSha256(data, privateKey); ```
 #
 # + input - The content to be signed
 # + privateKey - Private key used for signing
-# + return - The generated signature or `Error` if private key is invalid
+# + return - The generated signature or `crypto:Error` if private key is invalid
 public function signRsaSha256(byte[] input, PrivateKey privateKey) returns byte[]|Error = @java:Method {
     name: "signRsaSha256",
     class: "org.ballerinalang.stdlib.crypto.nativeimpl.Sign"
 } external;
 
 # Returns RSA-SHA384 based signature value for the given data.
+# ```ballerina byte[]|crypto:Error signature = crypto:signRsaSha384(data, privateKey); ```
 #
 # + input - The content to be signed
-# + privateKey - Private key used for signing.
-# + return - The generated signature or `Error` if private key is invalid
+# + privateKey - Private key used for signing
+# + return - The generated signature or `crypto:Error` if private key is invalid
 public function signRsaSha384(byte[] input, PrivateKey privateKey) returns byte[]|Error = @java:Method {
     name: "signRsaSha384",
     class: "org.ballerinalang.stdlib.crypto.nativeimpl.Sign"
 } external;
 
 # Returns RSA-SHA512 based signature value for the given data.
+# ```ballerina byte[]|crypto:Error signature = crypto:signRsaSha512(data, privateKey); ```
 #
 # + input - The content to be signed
 # + privateKey - Private key used for signing
-# + return - The generated signature or `Error` if private key is invalid
+# + return - The generated signature or `crypto:Error` if private key is invalid
 public function signRsaSha512(byte[] input, PrivateKey privateKey) returns byte[]|Error = @java:Method {
     name: "signRsaSha512",
     class: "org.ballerinalang.stdlib.crypto.nativeimpl.Sign"
 } external;
 
 # Verify RSA-MD5 based signature.
+# ```ballerina boolean|crypto:Error validity = crypto:verifyRsaMd5Signature(data, signature, publicKey); ```
 #
 # + data - The content to be verified
 # + signature - Signature value
 # + publicKey - Public key used for verification
-# + return - Validity of the signature or `Error` if public key is invalid
+# + return - Validity of the signature or `crypto:Error` if public key is invalid
 public function verifyRsaMd5Signature(byte[] data, byte[] signature, PublicKey publicKey)
                                       returns boolean|Error = @java:Method {
     name: "verifyRsaMd5Signature",
     class: "org.ballerinalang.stdlib.crypto.nativeimpl.Sign"
 } external;
 
 # Verify RSA-SHA1 based signature.
+# ```ballerina boolean|crypto:Error validity = crypto:verifyRsaSha1Signature(data, signature, publicKey); ```
 #
 # + data - The content to be verified
 # + signature - Signature value
 # + publicKey - Public key used for verification
-# + return - Validity of the signature or `Error` if public key is invalid
+# + return - Validity of the signature or `crypto:Error` if public key is invalid
```

Comment:
```suggestion
# + return - Validity of the signature or `crypto:Error` if public key is invalid
```
Please refer the template for the correct format.

Reply:
Corrected.